appengine_war.gradle

```diff
@@ -75,6 +75,12 @@ if (project.path == ":services:default") {
 
 appengine {
     deploy {
+        // appengineDeployAll task requires the version to be set. So, 
+        // this config lets gcloud select a version name when deploying 
```

Comment:
Isn't it better to always reuse the same version name for alpha? That way you don't have to deal with migrating traffic, or deleting old unused versions.